registry/repositories.go

```diff
@@ -4,14 +4,22 @@ type repositoriesResponse struct {
 	Repositories []string `json:"repositories"`
 }
 
-func (registry *Registry) Repositories() ([]string, error) {
+func (registry *Registry) Repositories() (repos []string, err error) {
```

Comment:
No need to use named return variables if you explicitely specify the variable names inside the function.